pyproject.toml

```
@@ -13,18 +13,18 @@ style = "pep440"
 dirty = true
 
 [tool.poetry]
-name = "[PACKAGE_NAME]"
+name = "[glass_box_umap]"
```

Our build system will register this package on PyPI using the value placed here. Thus, we want it to be:

```suggestion
name = "glass-box-umap"
```

---

Fixing in separate PR